store/migration/10002__add_vcs_github_com.sql

```diff
@@ -6,3 +6,5 @@ ALTER TABLE project_member ADD CONSTRAINT project_member_role_provider_check CHE
 
 ALTER TABLE vcs DROP CONSTRAINT vcs_type_check;
 ALTER TABLE vcs ADD CONSTRAINT vcs_type_check CHECK (type IN ('GITLAB_SELF_HOST', 'GITHUB_COM'));
+
+ALTER TABLE task ADD COLUMN blocked_by INTEGER ARRAY NOT NULL DEFAULT '{}';
```

Comment:
I think this should be done in a separate migration, otherwise, the file name is confusing